cscs-checks/microbenchmarks/mpi/src/halo_cell_exchange.c

```diff
@@ -0,0 +1,156 @@
+#include <stdlib.h>
```

Comment:
Please add a small description as of what this benchmark is doing exactly. Also, add comments in the code explaining briefly the different phases.